nf_core/launch.py

```diff
@@ -538,6 +538,22 @@ def validate_number(val):
 
             question["validate"] = validate_number
 
+            # Validate range type
+            def validate_range(val):
+                try:
+                    if val.strip() == "":
+                        return True
+                    fval = float(val)
+                    if "minimum" in param_obj and fval < float(param_obj["minimum"]):
+                        return "Must be greater than or equal to {}".format(param_obj["minimum"])
+                    if "maximum" in param_obj and fval > float(param_obj["maximum"]):
+                        return "Must be less than or equal to {}".format(param_obj["maximum"])
+                    return True
+                except ValueError:
+                    return "Must be a number"
+
+            question["validate"] = validate_range
```

Comment:
Don't we need some logic here to only use this validation if we have a min and a max? Otherwise it will just always overwrite the regular number validation function above..

Reply:
oh, that is true. How about if I completely replace the number validation function with the range validation (or add the min, max check to the number validation, which is the same at the end)? Because it does the same as the number validation, if there is no `min` or `max`.

Reply:
Yup 👍🏻

Reply:
There is already a test case for ranges in `test_launch.py`, if that is what you mean.